<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaseListWindow</class>
 <widget class="QMainWindow" name="CaseListWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>病歷查詢</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QTableWidget" name="tableWidget_medical_record_list">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <column>
         <property name="text">
          <string>CaseKey</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>註</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>門診日期</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>班別</string>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>完診</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>批價</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>診別</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>診號</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>病歷號</string>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>病患姓名</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>性別</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>出生日期</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>門診</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>負擔類別</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>就醫類別</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>卡序</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>療程</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>主治醫師</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>主診斷病名</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>給藥</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>推拿師</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>掛號費</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>門診負擔</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>藥品負擔</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>自費金額</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>檔案</string>
    </property>
    <addaction name="action_open_record"/>
    <addaction name="action_delete_record"/>
    <addaction name="separator"/>
    <addaction name="action_print_prescript"/>
    <addaction name="action_print_receipt"/>
    <addaction name="action_print_misc"/>
    <addaction name="separator"/>
    <addaction name="action_print_cases"/>
    <addaction name="action_print_fees"/>
    <addaction name="separator"/>
    <addaction name="action_export_cases_pdf"/>
    <addaction name="action_export_fees_pdf"/>
    <addaction name="separator"/>
    <addaction name="action_export_excel"/>
    <addaction name="separator"/>
    <addaction name="action_close"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>編輯</string>
    </property>
    <addaction name="action_set_check"/>
    <addaction name="action_set_uncheck"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="palette">
    <palette>
     <active/>
     <inactive/>
     <disabled/>
    </palette>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_requery"/>
   <addaction name="separator"/>
   <addaction name="action_open_record"/>
   <addaction name="action_delete_record"/>
   <addaction name="separator"/>
   <addaction name="action_print_prescript"/>
   <addaction name="action_print_receipt"/>
   <addaction name="separator"/>
   <addaction name="action_close"/>
  </widget>
  <action name="action_open_record">
   <property name="icon">
    <iconset>
     <normaloff>../icons/document-open.svg</normaloff>../icons/document-open.svg</iconset>
   </property>
   <property name="text">
    <string>調閱病歷</string>
   </property>
   <property name="toolTip">
    <string>調閱病歷</string>
   </property>
  </action>
  <action name="action_delete_record">
   <property name="icon">
    <iconset>
     <normaloff>../icons/user-trash.png</normaloff>../icons/user-trash.png</iconset>
   </property>
   <property name="text">
    <string>病歷刪除</string>
   </property>
  </action>
  <action name="action_print_prescript">
   <property name="icon">
    <iconset>
     <normaloff>../icons/document-print.svg</normaloff>../icons/document-print.svg</iconset>
   </property>
   <property name="text">
    <string>列印處方箋</string>
   </property>
  </action>
  <action name="action_requery">
   <property name="icon">
    <iconset>
     <normaloff>../icons/reload.svg</normaloff>../icons/reload.svg</iconset>
   </property>
   <property name="text">
    <string>重新查詢</string>
   </property>
   <property name="toolTip">
    <string>重新查詢病歷</string>
   </property>
  </action>
  <action name="action_close">
   <property name="icon">
    <iconset>
     <normaloff>../icons/application-exit.svg</normaloff>../icons/application-exit.svg</iconset>
   </property>
   <property name="text">
    <string>關閉視窗</string>
   </property>
  </action>
  <action name="action_print_receipt">
   <property name="icon">
    <iconset>
     <normaloff>../icons/printer-printing.png</normaloff>../icons/printer-printing.png</iconset>
   </property>
   <property name="text">
    <string>列印醫療收據</string>
   </property>
  </action>
  <action name="action_print_cases">
   <property name="icon">
    <iconset>
     <normaloff>../icons/printer.png</normaloff>../icons/printer.png</iconset>
   </property>
   <property name="text">
    <string>列印實體病歷</string>
   </property>
  </action>
  <action name="action_export_cases_pdf">
   <property name="icon">
    <iconset>
     <normaloff>../icons/document-export.svg</normaloff>../icons/document-export.svg</iconset>
   </property>
   <property name="text">
    <string>匯出實體病歷PDF</string>
   </property>
  </action>
  <action name="action_set_check">
   <property name="icon">
    <iconset>
     <normaloff>../icons/dialog-ok.svg</normaloff>../icons/dialog-ok.svg</iconset>
   </property>
   <property name="text">
    <string>全部註記</string>
   </property>
  </action>
  <action name="action_set_uncheck">
   <property name="icon">
    <iconset>
     <normaloff>../icons/button_cancel.svg</normaloff>../icons/button_cancel.svg</iconset>
   </property>
   <property name="text">
    <string>全不註記</string>
   </property>
  </action>
  <action name="action_print_fees">
   <property name="icon">
    <iconset>
     <normaloff>../icons/printer-printing.png</normaloff>../icons/printer-printing.png</iconset>
   </property>
   <property name="text">
    <string>列印收費明細</string>
   </property>
  </action>
  <action name="action_export_fees_pdf">
   <property name="icon">
    <iconset>
     <normaloff>../icons/document-export.svg</normaloff>../icons/document-export.svg</iconset>
   </property>
   <property name="text">
    <string>匯出收費明細PDF</string>
   </property>
  </action>
  <action name="action_export_excel">
   <property name="icon">
    <iconset>
     <normaloff>../icons/extract-archive.svg</normaloff>../icons/extract-archive.svg</iconset>
   </property>
   <property name="text">
    <string>匯出日報表 Excel</string>
   </property>
  </action>
  <action name="action_print_misc">
   <property name="icon">
    <iconset>
     <normaloff>../icons/printer-printing.png</normaloff>../icons/printer-printing.png</iconset>
   </property>
   <property name="text">
    <string>列印其他收據</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>pushButtonQueryClicked()</slot>
  <slot>pushButtonRequeryClicked()</slot>
  <slot>tableWidgetCellDoubleClicked()</slot>
  <slot>actionOpenCaseTriggered()</slot>
  <slot>actionDeleteCaseTriggered()</slot>
  <slot>actionPrintPresTriggered()</slot>
 </slots>
</ui>
